llvm/include/llvm/CodeGen/MachineDominators.h

```diff
@@ -44,11 +44,37 @@ extern template class DominatorTreeBase<MachineBasicBlock, true>; // PostDomTree
 using MachineDomTree = DomTreeBase<MachineBasicBlock>;
 using MachineDomTreeNode = DomTreeNodeBase<MachineBasicBlock>;
 
+namespace DomTreeBuilder {
+using MBBDomTree = MachineDomTree;
+using MBBUpdates = ArrayRef<llvm::cfg::Update<MachineBasicBlock *>>;
```

Comment:
Don't need llvm::?

Reply:
This is copied from `llvm/IR/Dominators.h`, and unqualified name lookup can find the proper declaration.